### ISSUE TITLE
Sperm flagella of irregular caliber

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#6504]
**Preferred term label:**
Sperm flagella of irregular caliber

**Synonyms**
Sperm flagella with irregular caliber

**Definition (free text, please give PubMed ID)**
See here for definitions
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7275805/

**Parent term (use hpo.jax.org/app)**
https://hpo.jax.org/app/browse/term/HP:0012868

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
OMIM:619095 
https://pubmed.ncbi.nlm.nih.gov/32681648/  1/1
https://pubmed.ncbi.nlm.nih.gov/32619401/  3/3

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
I think this is better with the primary label
Irregularly shaped sperm tail 
https://academic.oup.com/humrep/article/31/1/10/2379958

**Comment by pnrobinson** [comment]
added
